pkg/http_endpoint/authentication/jwt_authentication.go

```diff
@@ -0,0 +1,106 @@
+package httpAuth
+
+import (
+	"github.com/auth0/go-jwt-middleware"
+	"github.com/dgrijalva/jwt-go"
+	"net/http"
+	"time"
+	"visualization-api/pkg/http_endpoint/common"
+)
+
+const contextJWTProperty = "AuthToken"
+
+// JWTIssuer
+const tokenIssuer = "visualization-api"
+
+// CustomClaims defines what data would be stored in jwt token
+type CustomClaims struct {
+	IsAdmin   bool   `json:"is_admin"`
+	ProjectID string `json:"organizationId"`
+	jwt.StandardClaims
+}
+
+// JWTTokenFromParams creates jwt token given claims data
+func JWTTokenFromParams(secret string, isAdmin bool, projectID string,
+	expiresAt time.Time) (string, error) {
+
+	claims := CustomClaims{
+		isAdmin,
+		projectID,
+		jwt.StandardClaims{
+			ExpiresAt: expiresAt.Unix(),
```

Comment:
IntDate  A JSON numeric value representing the number of seconds from
      1970-01-01T0:0:0Z UTC until the specified UTC date/time.  See RFC
      3339 [RFC3339] for details regarding date/times in general and UTC
      in particular.

Is required format for date values according to https://tools.ietf.org/html/draft-ietf-oauth-json-web-token-15#section-4.1.4